Automattic/wp-calypso
# Instrument theme page for RUM

#### Changes proposed in this Pull Request

* Add theme page instrumentation for RUM performance tracking
* Make `lib/performance-tracking` SSR-safe, since the theme page is SSRed

#### Testing instructions

* Visit e.g. `/theme/barnsbury`
* Go to DevTools. You should see a request to `/logstash`. Verify there is sensible data being sent.
